tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -14,7 +14,7 @@ and limitations under the License.
 ***************************************************************************** */
 
 declare namespace ts {
-    const versionMajorMinor = "3.2";
+    const versionMajorMinor = "3.3";
```

Comment:
@DanielRosenwasser pls. Methinks you forgot to accept api baselines when you bumped versions last night. (Which is probably why the nightly publish failed)